### ISSUE TITLE
felix wiring: "wiringSenderRemoved" warnings

### ISSUE THREAD

**Issue by slintes** [INAETICS/kubernetes-demo-cluster#14]
```
2015.09.09 14:19:15 WARNING - Bundle: org.inaetics.remote.admin.wiring - Invocation of 'wiringSenderRemoved' failed. - java.util.ConcurrentModificationException
    at java.util.HashMap$HashIterator.nextNode(HashMap.java:1429)
    at java.util.HashMap$ValueIterator.next(HashMap.java:1458)
    at org.inaetics.remote.admin.wiring.RemoteServiceAdminImpl.wiringSenderRemoved(RemoteServiceAdminImpl.java:201)
    at org.inaetics.remote.admin.wiring.RemoteServiceAdminFactory.wiringSenderRemoved(RemoteServiceAdminFactory.java:69)
```

**Comment by slintes** [comment]
another one:

```
2015.09.09 14:27:27 WARNING - Bundle: org.inaetics.remote.admin.wiring - admin/wiring(453) wiringSenderRemoved: endpointWireId is null!
```

**Comment by slintes** [comment]
fixed in INAETICS/node-wiring-java@f2042da8d4efddd8016da822897691613aedbd89
